wasp/wasp.py

```diff
@@ -504,4 +507,27 @@ def schedule(self, enable=True):
 
         self._scheduling = enable
 
+
+    def set_theme(self, new_theme: bytes):
+        self._theme = array.array('H', new_theme)
+    def theme(self, theme_part: str) -> int:
+        """Returns the relevant part of theme. For more see ../tools/themer.py"""
+        theme_parts = ("ble",
+                       "scroll-indicator-up",
+                       "scroll-indicator-down",
+                       "battery-charging",
+                       "battery-empty",
+                       "battery-full",
+                       "status-clock",
+                       "confirm-yes",
+                       "confirm-no",
+                       "notify-icon",
+                       "watch-colon",
+                       "watch-left",
+                       "watch-right")
```

Comment:
Watch specific theme entries are not a great idea (we wouldn't want to give that to all applications). Should be probably be based on accent level: "accent-hi", "accent-lo", "accent-mid" instead so other apps can adopt if they wish.

Reply:
Also one of the more obvious things to theme is the slider widget and that is not here ;-)

Reply:
Slider has a `color` parameter, the only thing that can be changed maybe is the default color.